lib/Common/Core/EtwTraceCore.h

```diff
@@ -74,12 +74,18 @@ CompileAssert(false)
 #define MCGEN_PRIVATE_ENABLE_CALLBACK_V2(SourceId, ControlCode, Level, MatchAnyKeyword, MatchAllKeyword, FilterData, CallbackContext) \
        EtwCallback(ControlCode, CallbackContext)
 
+// Work-around for a bug in the instrumentationevents generator
+#pragma prefast(push)
+#pragma prefast(disable:__WARNING_USING_UNINIT_VAR, "The ETW data generated from the manifest includes a default null function which uses unintialized memory.")
```

Comment:
Is this because of how our manifest is structured, or would any project using ETW hit this same warning?

Reply:
I'm not entirely sure. I wasn't able to see a good way to fix it by changing the manifest, so I believe that it may be across many projects using ETW in a similar manner.